ante/msg_filter_ante.go

```diff
@@ -0,0 +1,35 @@
+package ante
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	ibcchanneltypes "github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
+)
+
+// MsgFilterDecorator defines an AnteHandler decorator for the v8 upgrade that
+// provide height-gated message filtering acceptance.
+type MsgFilterDecorator struct{}
+
+// AnteHandle performs an AnteHandler check that returns an error if the tx contains a message
+// that is blocked.
+// Right now, we block MsgTimeoutOnClose due to incorrect behavior that could occur if a packet is re-enabled.
```

Comment:
Is it possible to run into other issues from disabling this message?

Reply:
No, we'll be fine. Just forces any packet sent on that channel to have to wait until its timeout before it can be closed.